package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
Nope, it reruns on all changes. The option would be passed here: https://github.com/gabrielgrant/broccoli-less-single/pull/33/files#diff-168726dbe96b3ce427e7fedce31bb0bcR21